src/Native/System.Security.Cryptography.Native/CMakeLists.txt

```diff
@@ -26,6 +26,12 @@ if (OPENSSL_TLSV12)
     add_definitions(-DHAVE_TLS_V1_2=1)
 endif()
 
+include(CheckStructHasMember)
+check_struct_has_member ("SSL_CIPHER" algorithms openssl/ssl.h SSL_CIPHER_ALGORITHMS)
+if (SSL_CIPHER_ALGORITHMS)
+    add_definitions(-DHAVE_SSL_CIPHER_ALGORITHMS=1)
```

Comment:
I feel like other tests are checking for newer features, this one checks for an older one.